web/viewer.css

```diff
@@ -442,6 +442,7 @@ html[dir='rtl'] .findbar {
 }
 #findInput::placeholder {
   font-style: italic;
+  color: #cecece;
```

Comment:
Did you test this in Firefox, since there this change actually causes the placeholder text to be come less legible!?

Given that the issue you're attempting to fix is Chrome only, and that this change has a negative impact elsewhere, let's move this to this to here instead: https://github.com/mozilla/pdf.js/blob/6ac9e1c5ed0d5f067872b8482724c171c79566b2/web/viewer.css#L434

Also, let's use a colour in Chrome that's closer to the one used in Firefox, hence let's try `hsl(0, 0%, 75%)` instead (since the [standard colour](https://github.com/mozilla/pdf.js/blob/6ac9e1c5ed0d5f067872b8482724c171c79566b2/web/viewer.css#L1107) is defined using `hsl` notation).

Reply:
@Snuffleupagus , I'm new to open source , so I made a mistake by closing this pull request as I should have made changes to this only. I realised this mistake of mine and will keep this in mind for further contributions. As of now I have made the changes you had requested for and have opened a new pull request.